proof/refine/ARM/KHeap_R.thy

```diff
@@ -4288,4 +4288,52 @@ global_interpretation set_simple_ko: typ_at_pres "set_simple_ko C ptr ep"
 global_interpretation update_sk_obj_ref: typ_at_pres "update_sk_obj_ref C update ref new"
   unfolding typ_at_pres_def by wpsimp
 
+(* A standard (and active) scheduling context should have the following properties. They follow
+   from valid_sched_context' and sc_valid_refills as the following two lemmas show. *)
```

Comment:
Should there also be a lemma showing that `0 < scRefillMax sc'` follows from `sc_valid_refills sc`? And then maybe one putting them together and saying that `⟦valid_sched_context' sc' s'; sc_valid_refills sc; sc_relation sc n sc'⟧ ⟹ std_sc' sc'`?